### ISSUE TITLE
Plot Training Loss

### ISSUE THREAD

**Issue by xwasco** [alankbi/detecto#44]
**Describe the feature you'd like**
I would like to see the training loss not only the validation loss

**Describe the use cases of the feature**
To better detect cases of under/overfitting

**Additional context**

Thanks!

**Comment by alankbi** [comment]
Suggestion noted. In the meantime, a workaround could be to copy and modify the source code for the `fit` method to return both the training and validation losses.

**Comment by cagbal** [comment]
It would be really nice to have a progress bar as well.

**Comment by alankbi** [comment]
@cagbal just want to confirm: have you been using the `verbose=True` flag when training?

**Comment by cagbal** [comment]
Exactly. I trained the system with 10K images with bbs. It was hard to understand how long would one epoch take. A progress bar would help a lot.

**Comment by simeon9696** [comment]
@alankbi Would it be possible to link it to tensorboard?

**Comment by alankbi** [comment]
Maybe [this](https://pytorch.org/docs/stable/tensorboard.html) tutorial can help? With Detecto's Model class, you can call the `get_internal_model` method to get the actual PyTorch model, which you can then pass to the SummaryWriter as detailed in that tutorial.

**Comment by alankbi** [comment]
A progress bar has been added in [v1.1.2](https://github.com/alankbi/detecto/releases/tag/v1.2.0)
